### DISCUSSION TITLE
[412020]地铁离去

### DISCUSSION THREAD

**Discussion by mengqian-666** [pjy612/SteamManifestCache#114] [Share]
appid：412020
 密钥：
```
412021;f06003bf9aba2630c97f3f84b80f14067b0df94fb9bfabadcbaea93e1ae4c6f7
412022;fdaa0cc6fe8207b1742e3d1fe9b0c62fc848da589cba441eef409d816d6aa775
```
[地铁离去.zip](https://github.com/pjy612/SteamManifestCache/files/13751585/default.zip)
内测码：442CC2EFAD215B65
邮箱：sans_173@outlook.com
不知道行不行qwq

**Comment by mengqian-666** [comment]
qwq

**Reply by pjy612** [reply]
查收，以后请注意 app 和 密钥 的关系。
可以看下 项目内 Wiki
